lib/release_notes_bot_web/controllers/slack_interaction_controller.ex

```diff
@@ -16,10 +16,10 @@ defmodule ReleaseNotesBotWeb.SlackInteractionController do
           "#{user["name"]} has configured this channel to accept messages and updates for projects under: #{client_name}"
         )
 
-      %{client: client_name, project: project_name} ->
-        Channels.post_message(
-          Application.get_env(:release_notes_bot, :slack_channel),
-          "#{user["name"]} has created a new project for #{client_name} titled: '#{project_name}'"
+      %{client: client, project: project_name} ->
+        Channels.post_message_all_client_channels(
```

Comment:
This seems unrelated to `add repository url field to slack project modal `. This commit as a whole seems to be doing quite a few different things

Reply:
By adding the repo url field, it changed how the modal is parsed, and what is returned when it is parsed. When a new project modal is parsed, it now returns a client and its associate channels. Instead of pulling the slack channel from env, we now are able to take those channels and can broadcast a message to all client channels. I could break all this stuff up or rename the commit message to make everything more clear.